cloudpub/ms_azure/utils.py

```diff
@@ -147,25 +153,26 @@ def is_sas_eq(sas1: str, sas2: str) -> bool:
         log.debug("Got different base SAS: %s - Expected: %s" % (base_sas1, base_sas2))
         return False
 
-    # Parameters lengh differs
-    if len(params_sas1) != len(params_sas2):
-        log.debug(
-            "Got different lengh of SAS parameters: len(%s) - Expected len(%s)"
-            % (params_sas1, params_sas2)
-        )
-        return False
-
-    # Parameters values differs
-    for k, v in params_sas1.items():
-        if v != params_sas2.get(k, None):
-            log.debug("The SAS parameter %s doesn't match %s." % (v, params_sas2.get(k, None)))
+    if not base_only:
```

Comment:
I usually do not like much nested blocks

what about
```
if base_only:
    return True
```
But I am fine with current approach as well

Reply:
I thought about that as well, but it would be a bit similar with the `return True` from the function's last line.

Also I thought using an `elif` in the first statement and then add `return True` but I believe it would be not so legible.

If you have strong opinion on that I can change, but I considered 1 indentation tab to not be so bad from the readability perspective.

Reply:
One indentation level is not much issue. but it is added for bigger block (16 lines :-)

So you get to the indentation level 4 in the most nested block :-)
```
               log.debug("The SAS parameter %s doesn't match %s." % (v, params_sas2.get(k, None)))
                return False

    # Equivalent SAS
    return True
```

It is still OK to read and understand the code. 
Feel free to resolve and merge.